Copying diff url to clipboard

jcurbelo opened the pull request:
Using ZeroClipboard.js

pbu88 commented:
@jcurbelo UI wise this looks really great but we have to find a different way. ZeroClipboard uses flash and currently there's a lot of hate from browsers to flash. We'll need to find another way to copy it to the clipboard.

UI wise is perfect, although instead of putting in in the left, I would put it in the right next to the buttons, and maybe use the nice Bootstrap tooltip to show the "Copy URL" code.

jcurbelo commented:
I removed `ZeroClipBoard`, so we don't need to worry about the use of flash anymore :). 

About creating a "similar" clipboard library to GitHub's, i think is an excellent idea, since like you said @pbu88, it seems there is no other online, although if we rely on `document.execCommand('copy')` we need to consider the [browser's compability](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#Browser_Compatibility). But hey, just warn the users about keeping their browers updated and that's it.

I added some quite basics *copy to clipboard* functionalities into `clipboard.js`, just for the specific case of copying the url of the diff in question. But we can consider generic scenarios and then open source it, we might need `ZeroClipBoard` as a reference.

These are other things done:
- Added `boostrap tooltip` to **download**, **delete**, **copy to clipboard** buttons
- Putted UI Wise to the left

pbu88 commented:
Hey @jcurbelo thank you very much! This looks really good! Thanks for rebasing and cleaning the history.
